static/js/components/EducationForm.js

```diff
@@ -0,0 +1,285 @@
+import React from 'react';
+import Button from 'react-mdl/lib/Button';
+import IconButton from 'react-mdl/lib/IconButton';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+import Switch from 'react-mdl/lib/Switch';
+import FABButton from 'react-mdl/lib/FABButton';
+import Icon from 'react-mdl/lib/Icon';
+import Dialog from 'material-ui/Dialog';
+import { Card } from 'react-mdl/lib/Card';
+import _ from 'lodash';
+import moment from 'moment';
+
+import ProfileFormFields from '../util/ProfileFormFields';
+import { generateNewEducation } from "../util/util";
+import { saveAndContinue } from '../util/profile_edit';
+import { HIGH_SCHOOL } from '../constants';
+
+class EducationForm extends ProfileFormFields {
```

Comment:
so for the constructor here, I can move `educationLevelLabels` to be an instance property, which makes the constructor a little shorter, but I need to keep the `forEach` loop either here or in maybe `componentWillMount` so ensure it gets run before and rendering happens. 

what do you think? I think if we need to have the constructor for the `forEach` it doesn't really make sense to move the one thing to an instance property, but maybe it is a little cleaner that way.